regolith/utils.go

```diff
@@ -167,6 +171,11 @@ func getResolverCache(resolver string) (string, error) {
 	return getAppDataCachePath(appDataResolverCachePath, resolver)
 }
 
+// getResolverCache gets the dotRegolithPath from th app data folder
+func getFilterCache(url string) (string, error) {
+	return getAppDataCachePath(appDataFilterCachePath, url)
```

Comment:
I would print the info message about the returned path somewhere here in similar way to the `getAppDataDotRegolith` function below. This makes it easier for the user to locate the repository files when something goes wrong. This is useful especially because we're hashing the repository names to create the folder names.